src/Umbraco.Web/Security/BackOfficeSignInManager.cs

```diff
@@ -240,7 +240,7 @@ public new async Task<int> GetVerifiedUserIdAsync()
             {
                 return ConvertIdFromString(result.Identity.GetUserId());
             }
-            return -1;
+            return Constants.Security.SuperUserId - 1;
```

Comment:
Would it make more sense to return 0 here? which would indicate it's a non-user?

Reply:
@Shazwazza Wait until we have the migration from v7 to v8 ready.. ;-)